src/index.js

```diff
@@ -79,19 +80,20 @@ export default class ImageTool {
    * icon - Tool icon's SVG
    * title - title to show in toolbox
    *
-   * @return {{icon: string, title: string}}
+   * @returns {{icon: string, title: string}}
    */
   static get toolbox() {
     return {
       icon: ToolboxIcon,
-      title: 'Image'
+      title: 'Image',
     };
   }
 
   /**
-   * @param {ImageToolData} data - previously saved data
-   * @param {ImageConfig} config - user config for Tool
-   * @param {object} api - Editor.js API
+   * @param {object} tool - tool properties got from editor.js
```

Comment:
we used typedefs in JS to describe non-primitive types

```js
@typedef tool
@property ....
```

Reply:
it is just function parameters, because jsdoc plugin does not support destructuring assignment

https://github.com/gajus/eslint-plugin-jsdoc/issues/11